rust_src/src/xdisp.rs

```diff
@@ -0,0 +1,21 @@
+//! Display generation from window structure and buffer text.
+
+use remacs_macros::lisp_fn;
+
+use crate::remacs_sys::{trace_redisplay_p, EmacsInt};
+
+/// Toggle tracing of redisplay.
+/// With ARG, turn tracing on if and only if ARG is positive.
+#[cfg(feature = "glyph-debug")]
+#[lisp_fn(min = "0", intspec = "P")]
+pub fn trace_redisplay(arg: Option<EmacsInt>) {
```

Comment:
```suggestion
pub fn trace_redisplay(arg: InteractiveNumericPrefix) {
```

We have a type called `InteractiveNumericPrefix` which replaces what used to be `Fprefix_numeric_value`.

Reply:
Changed. Should now match original function more closely (adjusted tests).